02-peer-protocol.md

```diff
@@ -1272,7 +1272,7 @@ The handling of updates is similarly atomic: if the commit is not
 acknowledged (or wasn't sent) the updates are re-sent. However, it's not
 insisted they be identical: they could be in a different order,
 involve different fees, or even be missing HTLCs which are now too old
-to be added. Requiring they be identical would effectively mean a
+to be added. Requiring them being identical would effectively mean a
```

Comment:
NACK. either `Requiring that they be identical` or `Requiring them to be identical`